components/coachmark/stories/coachmark.stories.js

```diff
@@ -5,9 +5,9 @@ import metadata from "../dist/metadata.json";
 import packageJson from "../package.json";
 import { CoachMarkGroup } from "./coachmark.test.js";
 import {
-	Template,
-	CoachmarkMenuStatesTemplate,
 	CoachMarkMediaOptionsTemplate,
+	CoachmarkMenuStatesTemplate,
+	Template,
```

Comment:
My linter keeps correcting these.

Reply:
Ah yes, alphabetical sorting is in our linting system I think.